### PR TITLE
Add radix sort (least significant digit)

### PR DESCRIPTION
**Pull request by jdhrnndz**
Contributes to #2

### CODE REVIEW

**Comment by khoi** [comment]
@jdhrnndz thank you, I'll review it first thing tmr

File: algorithms.js [hunk 1]
```diff
@@ -211,3 +211,31 @@ function* shellSort(values) {
     }
   }
 }
+
+function* radixSortLSD(array) {
+  let values = array.slice(0, array.length);
+  let counter = Array.from({ length: 10 }).fill(0, 0);
```
Comment (khoi) [inline algorithms.js]: hmm how does this work? Can it be replaced by: 

```suggestion
  let counter = new Array(10).fill(0);
```

File: algorithms.js [hunk 2]
```diff
@@ -211,3 +211,31 @@ function* shellSort(values) {
     }
   }
 }
+
+function* radixSortLSD(array) {
+  let values = array.slice(0, array.length);
```
Comment (khoi) [inline algorithms.js]:
```suggestion
  let values = [...array];
```

File: algorithms.js [hunk 3]
```diff
@@ -211,3 +211,31 @@ function* shellSort(values) {
...
+    for (let i = 1; i < counter.length; i++) {
+      counter[i] = counter[i] + counter[i - 1];
+    }
+
+    for (let i = n - 1; i >= 0; i--) {
+      array[--counter[parseInt(values[i] / radix) % 10]] = values[i];
+      yield;
+    }
+
+    values = array.slice(0, array.length);
```
Comment (khoi) [inline algorithms.js]:
```suggestion
    values = [...array];
```

**Review by khoi — CHANGES_REQUESTED** [review]

**Comment by khoi** [comment]
@jdhrnndz Thank you for your contribution, just some small cosmetic change requested

File: sketch.js [hunk 4]
```diff
@@ -14,7 +14,8 @@ const SORT_CLASSES = {
   bogoSort: bogoSort,
   mergeSort: mergeSort,
   heapSort: heapSort,
-  shellSort: shellSort
+  shellSort: shellSort,
+  'radixSort (least significant digit)': radixSortLSD,
```
Comment (khoi) [inline sketch.js]:
```suggestion
  'radixSort (LSD)': radixSortLSD,
```

this will keep the select option width small

**Review by khoi — CHANGES_REQUESTED** [review]

**Comment by khoi** [comment]
@all-contributors please add @jdhrnndz for his code contribution

**Comment by jdhrnndz** [comment]
Alright, I will adjust the code accdg to these suggestions

**Review by khoi — APPROVED** [review]

**Comment by khoi** [comment]
@jdhrnndz can u also resolve the conflict, I don't have access to your repo so couldn't change it myself

**Comment by khoi** [comment]
Seems like your prettier config converts double quote to single, not a big deal. I'll make the change later. TY 🍻

**Comment by jdhrnndz** [comment]
> Seems like your prettier config converts double quote to single, not a big deal. I'll make the change later. TY 🍻

my bad. i did a hasty conflict resolution  😞

**Pull request merged**